codeflash/code_utils/tabulate.py

```diff
@@ -0,0 +1,3019 @@
+"""Copyright (c) 2011-2020 Sergey Astanin and contributors
```

Comment:
dont add the MIT license here directly. Codeflash is not MIT licensed. Say that This file is derived from so and so project which is available under MIT license.